### DISCUSSION TITLE
프로젝트 진행시 사용할 개발자 보드 관련하여 질문 있습니다.

### DISCUSSION THREAD

**Discussion by eking158** [Road-Balance/gcamp_ros2_basic#54] [Q&A]
현재 노트북을 통해 대부분의 연산 처리를 진행하고  최종적인 모터 제어 데이터만 산출하여 개발자 보드로 넘겨주는 형식의 로봇을 개발해 보고자 합니다. 
 이 과정에서 jetson nano 보드의 경우, 구하기도 힘들 뿐만 아니라 가격또한 매우 비쌉니다. 때문에 비교적 저렴한 가격에 구하기도 쉬운 '라즈베리파이 4 8gb'를 개발자 보드로 사용하고자 하는데, 혹시 충분한 성능이 나올까요??

개발자 보드의 역활은 노트북에서 넘겨받은 모터 데이터를 토대로 약 15개의 모터 제어를 진행하고, 카메라 영상을 노트북으로 공유해주는 역활 입니다.

**Comment by kimsooyoung** [comment]
해당 질문은 마지막 컨설팅 응답 시간에 일괄 답해드리도록 하겠습니다. 

컨설팅 구글 폼을 통해 더 자세한 내용을 알려주시면 응답에 도움이 될 것 같습니다
